opus/application/static_media/js/opus.js

```diff
@@ -151,6 +151,11 @@ var opus = {
 
     oldSurfacegeoTarget: null,
 
+    // Tooltipster settings
+    tooltips_max_width: 300,
+    tooltips_theme: "tooltipster-light",
+    tooltips_delay: 1500,
```

Comment:
I think this is pretty slow. Normal tooltips pop up a lot faster. Like maybe a half second? Is there any way to have the (i) info buttons show up fast and the mult values take longer?

Reply:
we can assign different delays for different tooltips. I modified it to have 1.5 second delay for mult tooltips and 0.5 second delay for the rest of tooltips.